### PR TITLE
Docs standardize

### PR DESCRIPTION
**Pull request by duartepinto**
This PR:
* Removes the Guides section in the docs website
* Adds the relevant external links (similar to https://android.developers.velocidi.com)

image

* Updates jazzy from 0.10.0 to 0.13.6

### CODE REVIEW

File: .jazzy.yaml [hunk 1]
```diff
@@ -10,15 +10,3 @@ module: VelocidiSDK
 output: ./docs/docs_page/
 theme: docs/velocidi-jazzy-theme/
 readme: ./docs/tmp/readme/README.md
```
Comment (thyandrecardoso) [inline .jazzy.yaml:12]: Why is this required?

File: .jazzy.yaml [hunk 2]
```diff
@@ -10,15 +10,3 @@ module: VelocidiSDK
...
-documentation: 
-  - ./docs/*.md 
-  - ./docs/tmp/docs/*.md 
-custom_categories:
```
Comment (thyandrecardoso) [inline .jazzy.yaml:16]: Do we still need the build steps that split the Readme?

File: .jazzy.yaml (hunk 1 above)
Comment (duartepinto) [reply .jazzy.yaml:12]: There has to be some kind of landing page. Thought about removing it, but it would default to just using the whole README as main page.

File: .jazzy.yaml (hunk 2 above)
Comment (thyandrecardoso) [reply .jazzy.yaml:16]: Ok, it seems we would still require this script for the `README` part that is used as the main page of the API documentation, and the rest of the partial files are used elsewhere on our documentation building pipelines. So, this is ok.

**Review by thyandrecardoso — APPROVED** [review]
